### ISSUE TITLE
Autostart help?

### ISSUE THREAD

**Issue by LittleCom** [Gibtnix/MSIKLM#38]
I have used your script for many years but only in Ubuntu 19.10 was I able to get the autostart script to work.
Today, I've installed Ubuntu 20.04.2 and for some reason, "sudo ./autostart.sh sudo msiklm green" will not work.

Could you please update the documentation so that we can see what they specify?
Could you also update the help file so that it actually helps us to fix this?

**Comment by Gibtnix** [comment]
Please try './autostart.sh <your Arguments>' but do not add 'sudo' or 'msiklm' here and let me know if it works for you. If not, I'll try to find a solution.

**Comment by LittleCom** [comment]
Thank you for providing this suggestion, it works.
For others who might come across this, here are some helpful details:

Laptop: MSI GT ONG (Workstation with K2000 GFX chip)

After installing the msiklm script, we can turn on the backlit keyboard with:
sudo msiklm green (substitute green for any color specified in the help file)

Autostart:
On Ubuntu 20.10, I issued: sudo ./autostart.sh msiklm green (and it worked).
Ubuntu LTS 20.04.2 seems to need what Gibtnix suggested:
./autostart.sh green (again, pick your favorite color), enter admin pw and enjoy.

I have tested this, rebooted and consider this issue solved.  
Lastly, a big thank you to Gibtnix for making this script.  This laptop is old but sill OK to use especially with Linux. Thanks to his MSIKLM script, typing in a dark room or at least positioning the fingers is quick and easy. 
Thank you! :)
